lib/manageiq/api/common/config_editor.rb

```diff
@@ -0,0 +1,21 @@
+module ManageIQ
+  module API
+    module Common
+      class ConfigEditor
```

Comment:
Shouold this be more specific and include "Metrics" in the name?

Reply:
I was intending to add other methods for things that alter the rails application config object. The alternative approach is to have a bunch of classes that look like this one, but if we do runtime requires for all of the different features it still remains "opt-in" this way and it doesn't look like there will be too many extra helpers and such.

Reply:
Oh, I was thinking something more along the lines of calling something like this in the engine definition
```ruby
ManageIQ::API::Common::Logging.install
ManageIQ::API::Common::Metrics.install
```

Reply:
Yeah, I'm fine with either. As I started writing it it seemed like overkill, but I can go that direction if you feel strongly about it. @Fryguy any opinions here?